service/rpi-sb-provisioner.sh

```diff
@@ -932,7 +932,7 @@ announce_stop "Writing OS images"
 
 if [ -z "${DEMO_MODE_ONLY}" ] && [ -d "${RPI_DEVICE_RETRIEVE_KEYPAIR}" ]; then
     announce_start "Capturing device keypair to ${RPI_DEVICE_RETRIEVE_KEYPAIR}"
-    get_variable private-key > "${RPI_DEVICE_RETRIEVE_KEYPAIR}/${TARGET_DEVICE_SERIAL}.der"
+    get_variable private-key > "${RPI_DEVICE_RETRIEVE_KEYPAIR}/${TARGET_DEVICE_SERIAL}.der" || true
```

Comment:
This will fail silently - but I'd prefer a warning in the log to indicate this happened.

Among other things - this would be a good way of identifying supply chain attacks where a malicious actor has pre-written the customer OTP rows.

Reply:
Would a keywriter_log here be appropriate or shall we just stick it in provisioner_log?

Reply:
Keywriter log would be best, I think.